ENGAGEHF/Education/Views/EducationalVideoCard.swift

```diff
@@ -0,0 +1,61 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct EducationalVideoCard: View {
+    let video: Video
+    
+    
+    var body: some View {
+        ZStack(alignment: .bottom) {
+            ThumbnailView(youtubeId: video.youtubeId)
+            
+            HStack {
+                VStack(alignment: .leading) {
+                    Text(video.title)
+                        .font(.title3.bold())
+                        .foregroundStyle(.white)
+                        .lineLimit(1)
+                    Text(video.description ?? "Video Description")
+                        .font(.headline.bold())
+                        .foregroundStyle(.white.opacity(0.75))
+                        .lineLimit(1)
```

Comment:
I would suggest to not render the video description `Text` if there is no description (either nil or an empty string) for a video using an `if let` statement here.